extensions/audio-device/src/audio-device.ts

```diff
@@ -0,0 +1,141 @@
+import path from "path";
+import { execa } from "execa";
+import { environment } from "@raycast/api";
+import fs from "fs";
+
+enum TransportType {
+  avb,
+  aggregate,
+  airplay,
+  autoaggregate,
+  bluetooth,
+  bluetoothle,
+  builtin,
+  displayport,
+  firewire,
+  hdmi,
+  pci,
+  thunderbolt,
+  usb,
+  virtual,
+  unknown,
+}
+
```

Comment:
```suggestion
```
Could be removed, seems obsolete?

Reply:
It's using an integer value right now, but macos-audio-devices returns a `string` here. I improved this a bit and make use of the name instead of the value to show as a subtitle.